src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -36,10 +36,27 @@
     validate_set_secret,
     validate_retention_days,
     validate_registry_name,
-    validate_expiration_time
+    validate_expiration_time,
+    validate_manifest_id,
+    validate_repo_id,
+    validate_repository
 )
 from .scope_map import RepoScopeMapActions, GatewayScopeMapActions
 
+repo_id_type = CLIArgumentType(
+    nargs='*',
+    default=None,
+    validator=validate_repo_id,
+    help="A fully qualified repository specifier such as 'MyRegistry.azurecr.io/hello-world'."
+)
+
+manifest_id_type = CLIArgumentType(
+    nargs='*',
+    default=None,
+    validator=validate_manifest_id,
```

Comment:
this is cool I didn't know we could add built-in validators for the parameters. Fancy (y)